Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use an official Python runtime as a parent image
```

Comment:
I'll have to refresh my Docker hub account probably. 😀

Reply:
I'll push it after merge to [Docker Hub parallelo3301/zulip-chatgpt-bot](https://hub.docker.com/repository/docker/parallelo3301/zulip-chatgpt-bot). Might add some GitHub Actions so the image is not out-of-date.